src/app/create-message.ts

```diff
@@ -0,0 +1,3 @@
+export class CreateMessage {
```

Comment:
It's better to use an interface than a class here.

Reply:
done.